exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if(mark >= 80){
+    return "A";
+  } else if (mark < 80 && mark > 60) {
+    return "B";
+  } else if (mark <= 60 && mark >50) {
```

Comment:
my explanation to '...no lower than 50...' that is should be cover "equal and greater than 50"

Reply:
Thank you very much for your suggestion. I will amend my code.